src/common/helper.js

```diff
@@ -543,20 +552,26 @@ function * checkGetAccess (authUser, submission, parentSpan) {
         const screener = _.filter(currUserRoles, { role: 'Primary Screener' })
         const reviewer = _.filter(currUserRoles, { role: 'Reviewer' })
 
-        // User is either a Reviewer or Screener
-        if (screener.length !== 0 || reviewer.length !== 0) {
+        if (screener.length !== 0) { // User is Screener
           const screeningPhase = _.filter(phases, { phaseType: 'Screening', phaseStatus: 'Scheduled' })
+
+          // Screening is not Opened / Closed
+          if (screeningPhase.length !== 0) {
```

Comment:
@sharathkumaranbu I came across a challenge in the dev environment which did not have a `Screening` phase at all. The current logic will allow any `Primary Screener` to view the submissions in such a case.

Is this a valid case in the production environment (challenge having no screening phases)? If so, the code needs to be updated to handle that.

Reply:
@meshde Earlier we had Prototype and Assembly challenges which had screening phases, hence we had to handle Screener role as well for Development challenges. Right now only Design challenges have screening phase.

Reply:
The code has been updated to specifically check for Open and Closed statuses for Screening phases.